### PR TITLE
Named exports from ol/css module

### PR DESCRIPTION
**Pull request by marcjansen**
Title says it all.

Please review.

### CODE REVIEW

**Comment by marcjansen** [comment]
Makes sense, will see how and when I can update this.

**Comment by marcjansen** [comment]
@ahocevar updated as requested. This is ready for review again.

I was thinking of using `const`, but we do not use it elsewhere, right? E.g.:

```js
// instead of …
export var CLASS_CONTROL = 'ol-control';
// … we could use const
export const CLASS_CONTROL = 'ol-control';
```

 If you want, I can update accordingly

File: src/ol/css.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ _ol_css_.CLASS_CONTROL = 'ol-control';
  * @param {string} The CSS font property.
  * @return {Object.<string>} The font families (or null if the input spec is invalid).
  */
-_ol_css_.getFontFamilies = (function() {
+export var getFontFamilies = (function() {
```
Comment (fredj) [inline src/ol/css.js]: `export function` instead of `export var` ?

Comment (marcjansen) [reply src/ol/css.js]: I stumbled over this as well, here we are using an IIFE, so we can cache certain things. I wasn't sure how to handle this. simpy changing as you suggested raises an 'Expression expected' error. What do you suggest?

IMO `var` isn't technically false, is it?

Comment (marcjansen) [reply src/ol/css.js]: Since things are passing with this, I am tempted to merge this.

Comment (fredj) [reply src/ol/css.js]: yes, please merge

**Comment by marcjansen** [comment]
Thanks for the reviews, @ahocevar & @fredj

**Pull request merged**